lib/cartopy/tests/mpl/test_images.py

```diff
@@ -179,6 +179,15 @@ def test_stock_img():
     ax.stock_img()
 
 
+@pytest.mark.xfail((5, 0, 0) <= ccrs.PROJ4_VERSION < (5, 1, 0),
+                   reason='Proj Orthographic projection is buggy.',
+                   strict=True)
+@ImageTesting(['imshow_etopo1_ortho'], tolerance=0.7)
```

Comment:
It appears that you haven't added this test image?

Reply:
Done.